package.json

```
@@ -26,5 +26,8 @@
   "lint-staged": {
     "*.{ts,js,md}": "prettier --write"
   },
-  "packageManager": "yarn@4.4.0"
+  "packageManager": "yarn@4.4.0",
+  "dependencies": {
+    "vite": "6.0.0"
+  }
```

This is not needed. The dependency needs to be updated in frontend workspace of the monorepo.
```suggestion
  "packageManager": "yarn@4.4.0"
```

https://github.com/aws-samples/aws-sdk-js-notes-app/blob/672ced4db2b9a2a316e15f15f3bfb04262c58c43/packages/frontend/package.json#L64